roles/filetree_create/templates/current_user_access_list.j2

```diff
@@ -0,0 +1,29 @@
+{% if first_user_role | default(true) | bool %}
+---
+controller_roles:
+{% endif %}
+    users:
```

Comment:
if you want to use this exported controller_roles afterwards with dispatch role, it will have to be a list, not a dict.